<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddBook</class>
 <widget class="QDialog" name="AddBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Book</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>571</width>
     <height>631</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="Holder">
    <item>
     <layout class="QHBoxLayout" name="InnerHolder">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QGridLayout" name="Form">
        <item row="2" column="0">
         <widget class="QLabel" name="GenreATag">
          <property name="text">
           <string>Genre 1:</string>
          </property>
          <property name="buddy">
           <cstring>GenreALine</cstring>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="dateTag">
          <property name="text">
           <string>Date added:</string>
          </property>
          <property name="buddy">
           <cstring>dateLine</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="GenreBLine"/>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="GenreALine"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="TitleTag">
          <property name="text">
           <string>Book Title:</string>
          </property>
          <property name="buddy">
           <cstring>TitleLine</cstring>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="dateLine"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="IdTag">
          <property name="text">
           <string>Id Number:</string>
          </property>
          <property name="buddy">
           <cstring>IdLine</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="TitleLine"/>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="AuthorLine"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="AuthorTag">
          <property name="text">
           <string>Author Name:</string>
          </property>
          <property name="buddy">
           <cstring>AuthorLine</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="IdLine"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="GenreBTag">
          <property name="text">
           <string>Genre 2:</string>
          </property>
          <property name="buddy">
           <cstring>GenreBLine</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="SaveBtn">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>TitleLine</tabstop>
  <tabstop>AuthorLine</tabstop>
  <tabstop>GenreALine</tabstop>
  <tabstop>GenreBLine</tabstop>
  <tabstop>IdLine</tabstop>
  <tabstop>dateLine</tabstop>
  <tabstop>SaveBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
